migrations/versions/2019_02_12_new_mat_views.py

```diff
@@ -0,0 +1,202 @@
+"""Create new materialized views for updated database model.
+
+This migration only adds new materialized views - it leaves the existing ones in tact to be removed by a later
+migration/PR.
+
+Revision ID: 2019_02_12_new_mat_views
+Revises: 2019_01_15_uq_measure_version
+Create Date: 2019-01-15 12:25:28.892266
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = "2019_02_12_new_mat_views"
+down_revision = "2019_01_15_uq_measure_version"
+branch_labels = None
+depends_on = None
+
+latest_published_measure_versions_view = """
+CREATE MATERIALIZED VIEW new_latest_published_measure_versions AS
```

Comment:
Feels a bit icky to have to add the `new_` prefix... 😬   Do you intend to drop it later after deleting the old materialised views?

Reply:
Yeah, this will have some follow-up work to drop all the old materialized views and then rename these to drop the `new_` prefix. It's in the checklist already for the `tidy up stuff` ticket.

Reply:
👍